test/klippy/printers.test

```diff
@@ -258,6 +258,7 @@ CONFIG ../../config/generic-bigtreetech-skr-pico-v1.0.cfg
 # Anycubic Printers using trigorilla board with the hc32f460
 DICTIONARY hc32f460-serial-PA3PA2.dict
 CONFIG ../../config/printer-anycubic-kobra-plus-2022.cfg
+CONFIG ../../config/printer-anycubic-kobra-go-2022.cfg
```

Comment:
This should be in alphabetical order.

Thanks

Reply:
It should be good now.
Thank you